docs/index.rst

```diff
@@ -86,7 +100,8 @@ Setting                Default                Description
 ======================    =================      ==================================================================
 Setting                   Default                Description
 ======================    =================      ==================================================================
-**gcloud.credentials**    **required**           Path to the Service Accounts credentials JSON file.
+**gcloud.credentials**                           Path to the Service Accounts credentials JSON file.
+**gcloud.project**                               **required** if running without a credentials file and in an environment where the project can't automatically be determined.
```

Comment:
the `project` argument is not a strict requirement, e.g., when running on GKE, `google-cloud-storage` can determine the project on it's own, when running e.g. locally, it can't, so I've tried to say something like `**required**, if` here :thinking: